From 37e57803e7c38c8483247e5499076120a29eb447 Mon Sep 17 00:00:00 2001
From: Lin Ruizhe <linruizhe@huawei.com>
Date: Fri, 31 Dec 2021 15:59:25 +0800
Subject: [PATCH] irqdomain: Add kabi_reserve in irqdomain
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 37e57803e7c38c8483247e5499076120a29eb447


hulk inclusion
bugzilla: https://gitee.com/openeuler/kernel/issues/I4MZU1
CVE: NA

---------------------------

Add kabi_reserve in irqdomain.h

Signed-off-by: Lin Ruizhe <linruizhe@huawei.com>
Reviewed-by: Liao Chang <liaochang1@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 include/linux/irqdomain.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/irqdomain.h b/include/linux/irqdomain.h
index ea5a337e0f8b..3f493c858fc6 100644
--- a/include/linux/irqdomain.h
+++ b/include/linux/irqdomain.h
@@ -35,6 +35,7 @@
 #include <linux/of.h>
 #include <linux/mutex.h>
 #include <linux/radix-tree.h>
+#include <linux/kabi.h>
 
 struct device_node;
 struct irq_domain;
@@ -184,6 +185,7 @@ struct irq_domain {
 	unsigned int revmap_size;
 	struct radix_tree_root revmap_tree;
 	struct mutex revmap_tree_mutex;
+	KABI_RESERVE(1)
 	unsigned int linear_revmap[];
 };
 
-- 
2.26.2

